pattern_library/monkey_utils.py

```diff
@@ -9,7 +9,7 @@
 logger = logging.getLogger(__name__)
 
 
-def override_tag(register, name):
+def override_tag(register, name, default=None):
```

Comment:
Could we be more descriptive than `default` with the name here? I think something like `default_html` would better reflect the fact that this is specifying markup as a string.

Additionally, we need a sentinel value (e.g. `UNSPECIFIED = object()`) as the default for this argument and the check on L79 needs to be `default is not UNSPECIFIED` or we'll never be able to set falsey values as the default output of tags.

Reply:
Interesting, I did not consider this! Thanks.